arviz/tests/external_tests/test_data_pyjags.py

```diff
@@ -0,0 +1,147 @@
+# pylint: disable=redefined-outer-name
+
+import typing as tp
+import numpy as np
+import pytest
+
+import arviz as az
+
+from arviz.data.io_pyjags import (
+    _convert_pyjags_samples_dictionary_to_arviz_samples_dictionary,
+    _convert_arviz_samples_dictionary_to_pyjags_samples_dictionary,
+    _extract_samples_dictionary_from_arviz_inference_data,
+    from_pyjags,
+)
+
+
+@pytest.fixture()
+def pyjags_samples_dict() -> tp.Dict[str, np.ndarray]:
+    return {
+        "b": np.array(
```

Comment:
`np.random.normal(size=(...))` can be used to reduce the lines used

Reply:
Good idea, I incorporated this into the tests.